api/proto/store/v1alpha2/sync_service.proto

```diff
@@ -0,0 +1,138 @@
+// Copyright AGNTCY Contributors (https://github.com/agntcy)
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package store.v1alpha2;
+
+import "google/protobuf/empty.proto";
+import "store/v1alpha2/object.proto";
+
+// SyncService provides functionality for synchronizing objects between Directory nodes.
+// 
+// This service enables one-way synchronization from a remote Directory node to the local node,
+// allowing distributed Directory instances to share and replicate objects. The service supports
+// both on-demand synchronization and tracking of sync operations through their lifecycle.
+service SyncService {
+  // CreateSync initiates a new synchronization operation from a remote Directory node.
+  //
+  // The operation is non-blocking and returns immediately with a sync ID that can be used
+  // to track progress and manage the sync operation.
+  //
+  // Returns: A unique sync ID string for tracking the operation
+  rpc CreateSync(CreateSyncRequest) returns (CreateSyncResponse);
+
+  // ListSyncs returns a stream of all sync operations known to the system.
+  //
+  // This includes active, completed, and failed synchronizations.
+  //
+  // Returns: A stream of sync IDs for all known synchronizations
+  rpc ListSyncs(google.protobuf.Empty) returns (stream ListSyncItem);
```

Comment:
```suggestion
  rpc ListSyncs(ListSyncsRequest) returns (ListSyncsResponse);
```

i would consider switching here to a paginated result like search or change the `ListSyncItem` to `ListSyncItems` that can return multiple items